extensions/WSLExtension/DistributionDefinitions/DistributionDefinition.yaml

```diff
@@ -0,0 +1,74 @@
+# This yaml file is used to map a distributions name to it logo file in the WslAssets folder. These distribution names

```

Comment:
NIT: Remove `This YAML file is` I believe developers will know they are looking at a yaml file.

Reply:
Removed, thanks